nwbib/nwbib-spatial.ttl

```diff
@@ -26148,7 +26148,7 @@ nwbib-spatial:Q876515
         a               skos:Concept ;
         skos:broader    nwbib-spatial:N12 ;
         skos:inScheme   <https://nwbib.de/spatial> ;
-        skos:prefLabel  "Minden-Ravensberg"@de ;
+        skos:prefLabel  "Minden-Ravensberg (bis 1807)"@de ;
```

Comment:
Hmm, mit der Ergänzung des Enddatums werden die nicht so zufrieden sein. Überhaupt ist die Wikipedia-Verknüpfung an dieser Stelle fragwürdig. In Wikipedia ist es eine historische Verwaltungseinheit in der NWBib eine Landschaft...